src/events/crawler/lib/fetch.js

```diff
@@ -55,15 +55,18 @@ export const fetch = async (url, type, date = process.env.SCRAPE_DATE || datetim
     }
 
     const response = await needle('get', url);
-    const fetchedBody = toString ? response.body.toString() : response.body;
-
-    await caching.saveFileToCache(url, type, date, fetchedBody);
 
     if (disableSSL) {
       delete process.env.NODE_TLS_REJECT_UNAUTHORIZED;
     }
 
-    return fetchedBody;
+    if (response.statusCode < 400) {
```

Comment:
or do we just want 200s? (how to handle redirects? will they already be handled before this path?)